webui-src/app/people/people_ownids.js

```diff
@@ -132,7 +132,7 @@ const Identity = () => {
           details = data.details;
           // Creating URI during fetch because `details` is uninitialized
           // during view run, due to request being async.
-          avatarURI = 'data:image/png;base64,' + data.details.mAvatar.mData;
+          avatarURI = data.details.mAvatar.mData.base64 == '' ? '' : 'data:image/png;base64,' + data.details.mAvatar.mData.base64;
```

Comment:
I think it would be better to have strict checks when dealing with empty strings:
```suggestion
          avatarURI = data.details.mAvatar.mData.base64 === '' ? '' : 'data:image/png;base64,' + data.details.mAvatar.mData.base64;
```